tests/scripts/run-xunit-perf.cmd

```diff
@@ -142,6 +141,17 @@ rem ****************************************************************************
     shift
     goto :parse_command_line_arguments
   )
+  IF /I [%~1] == [-warmup] (
```

Comment:
If this is not default, then it should be added in the `perf.groovy` file to the CQ/JIT benchmark runs, so they keep their current behavior.